games/ltris2/Portfile

```diff
@@ -0,0 +1,51 @@
+PortSystem              1.0
+PortGroup               app 1.0
+
+name                    ltris2
+version                 2.0.4
+categories              games
+maintainers             nomaintainer
+license                 GPL-2+
+use_parallel_build      yes
+
+description             Tetris clone with multiplayer and AI support.
+long_description        LTris as a tetris clone which means you have a bowl \
+                        with blocks falling down. By rotating and moving the \
+                        blocks you try to assemble whole lines which then \
+                        disappear. LTris has three modes for this: Classic is \
+                        the classical one where you play until the bowl \
+                        becomes filled, Figures resets the bowl contents to a \
+                        new figure for each level and adds suddenly appearing \
+                        tiles and lines later on and Multiplayer where up to \
+                        three players either controlled by human or CPU(!) \
+                        compete and send completed lines to each other.
+
+homepage                https://lgames.sourceforge.net/index.php?project=LTris
+master_sites            sourceforge:project/lgames/ltris
+
+checksums               rmd160  0b7133a36c21077817184c8166be52f6548a8237 \
+                        sha256  482150495f9d87bb139d5af1abec7030383c37fdb3e75a45e9bad67ead798eda \
+                        size    2944544
+
+depends_lib             port:gettext \
+                        port:libsdl2 \
+                        port:libsdl2_ttf \
+                        port:libsdl2_image \
+                        port:libsdl2_mixer
+post-extract {
+    move ${worksrcpath}/src/sdl.h ${worksrcpath}/src/sdl_.h
+}
+
+configure.ldflags-append \
+                        "-lintl"
+
+patchfiles              patch-sdl_.h.diff \
+                        patch-install-mode.diff
+
+configure.args          --with-highscore-path=${prefix}/var/lib/games/ \
```

Comment:
Is this argument actually supported? In v. 1 it is not, despite sitting in the portfile.